soultion3.txt

```diff
@@ -0,0 +1,31 @@
+bca_info = """
+BCA (Bachelor of Computer Applications) Course Information:
+
+BCA is an undergraduate degree program in the field of computer applications. It is a popular choice for students who are interested in the world of computers and programming. The course covers subjects like programming languages, data structures, algorithms, software development, and more. BCA graduates can pursue careers in software development, web development, database management, and other related fields.
+
+Duration: 3 years
+Eligibility: 10+2 with Mathematics
+
+For more information, visit: www.example.com/bca
+"""
+
+with open("BCA.txt", "w") as bca_file:

```

Comment:
why you write "with open" in program?

Reply:
"with open" is use to open file.
DO not close the program.